flexmeasures/data/schemas/reporting/aggregation.py

```diff
@@ -0,0 +1,67 @@
+from marshmallow import fields, ValidationError, validates_schema
+
+from flexmeasures.data.schemas.reporting import ReporterConfigSchema
+
+from enum import Enum
+
+
+class AggregationMethod(Enum):
+    SUM = "sum"
+    MEAN = "mean"
+
+
+class AggregatorSchema(ReporterConfigSchema):
+    """Schema for the reporter_config of the AggregatorReporter
+
+    Example:
+    .. code-block:: json
+        {
+            "beliefs_search_configs": [
+                {
+                    "sensor": 1,
+                    "source" : 1,
+                    "alias" : "PV"
+                },
+                {
+                    "sensor": 1,
+                    "source" : 2,
+                    "alias" : "CONSUMPTION"
+                }
+            ],
+            "method" : "SUM",
```

Comment:
I'm okay-ish with the capitalization in the aliases, but the capitalized method aliases seem like an unnecessary break from other reporter_config syntax.

Reply:
Sure! Also, I've ended up going lower case everywhere... :S